src/plugins/presentation_util/kibana.json

```diff
@@ -10,6 +10,6 @@
   "server": true,
   "ui": true,
   "extraPublicDirs": ["common/lib"],
-  "requiredPlugins": ["savedObjects", "kibanaReact", "embeddable", "expressions", "dataViews"],
```

Comment:
It caused a circular dependency, so I copied the **deprecated** `SavedObjectSaveModal`